SU-HSDO/suhumsci
# Remove legacy themes modules

# READY FOR REVIEW

## Summary
- Remove legacy themes
- Update documentation to remove references to legacy themes

## Need Review By (Date)
07/12

## Urgency
medium

## Steps to Test
1. Do a manual inspect of both colorful and traditional site. Confirm that everything works as expected

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
